applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c

```diff
@@ -12,36 +12,33 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(le_audio, CONFIG_BLE_LOG_LEVEL);
 
-int le_audio_frame_create(struct audio_data *audio_frame, const struct bt_bap_stream *stream,
-			  const struct bt_iso_recv_info *info, const struct net_buf *buf)
+int le_audio_metadata_populate(struct audio_metadata *meta, const struct bt_bap_stream *stream,
+			       const struct bt_iso_recv_info *info, const struct net_buf *buf)
 {
 	int ret;
 
 	/* Populate the audio frame structure */
-	audio_frame->data = buf->data;
-	audio_frame->data_size = buf->len;
-	audio_frame->meta.bad_data = false;
-	audio_frame->meta.reference_ts_us = info->ts;
-	le_audio_freq_hz_get(stream->codec_cfg, &audio_frame->meta.sample_rate_hz);
-	le_audio_duration_us_get(stream->codec_cfg, &audio_frame->meta.data_len_us);
-
-	ret = bt_audio_codec_cfg_get_chan_allocation(stream->codec_cfg,
-						     &audio_frame->meta.locations, true);
+	meta->bad_data = false;
+	meta->reference_ts_us = info->ts;
+	le_audio_freq_hz_get(stream->codec_cfg, &meta->sample_rate_hz);
+	le_audio_duration_us_get(stream->codec_cfg, &meta->data_len_us);
```

Comment:
See here that we should rename 
```suggestion
	le_audio_duration_us_get(stream->codec_cfg, &meta->data_dur_us);
```

Reply:
Discuss changing the name of struct members that are in `nrf/include`

Reply:
We will take this as a separate discussion